keras/src/layers/preprocessing/image_preprocessing/random_flip.py

```diff
@@ -93,8 +96,45 @@ def transform_bounding_boxes(
         bounding_boxes,
         transformation,
         training=True,
+        input_shape=None,
     ):
-        raise NotImplementedError
+        flips = transformation["flips"][0]
+        input_height, input_width = input_shape
```

Comment:
Please put that information in the `transformation` dict instead

Reply:
I put this value to transformation['input_shape']